docs/HowToGuides/GettingStarted.md

```diff
@@ -253,41 +253,30 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
 
 ### The actual build
 
-1. Build the toolchain with optimizations, debuginfo, and assertions, using
-   Ninja.
-   - macOS:
-     ```sh
-     utils/build-script --skip-build-benchmarks \
-       --skip-ios --skip-watchos --skip-tvos --swift-darwin-supported-archs "$(uname -m)" \
-       --sccache --release-debuginfo --swift-disable-dead-stripping
-     ```
-   - Linux:
-     ```sh
-     utils/build-script --release-debuginfo
-     ```
-     If you installed and want to use Sccache, include the `--sccache` option in
-     the invocation as well.
-   <!-- FIXME: Without this "hard" line break, the note doesn’t get properly spaced from the bullet -->
-   <br />
+Build the toolchain with optimizations, debuginfo, and assertions, using Ninja:
 
-   > **Note**  
-   > If you are planning to work on the compiler on macOS, but not the parts that are
-   > written in Swift, pass `--bootstrapping=hosttools` to speed up local
-   > development.
```

Comment:
It would be nice if the converse of this was kept, ie telling people working on the Swift parts what to do.

Reply:
The converse is simply the default (not to supply the mode). But I suspect very few people actually use 2-stage bootstrapping for development because the editing cycle is a lot slower. Personally, I would rather rely on CI to catch any instances of a just-built compiler not being happy with whatever I did using my host compiler.

Reply:
We should make `hosttools` the default mode

Reply:
[It already is for linux, _if_ CMake can find a prebuilt Swift compiler](https://github.com/apple/swift/blob/main/CMakeLists.txt#L875), don't know about macOS.

Reply:
Sneaky! That’s even better. macOS default is `bootstrapping-with-hostlibs`.

Reply:
Oh I should have commented in here. The default is changing - https://github.com/apple/swift/pull/70343. I have some more checks to run, but I expect to get that in Soon (TM)

Reply:
> The converse is simply the default (not to supply the mode).

Right, I'm saying that it would be nice if this note were changed to address that instead, since more and more of the compiler is being written in Swift.

Reply:
> Right, I'm saying that it would be nice if this note were changed to address that instead, since more and more of the compiler is being written in Swift.

Ah, but since `hosttools` is the mode we want to recommend for working on the Swift parts, there is nothing else be done (both the Linux and macOS commands will produce a `hosttools` build system). Am I missing the point?

Reply:
The note you are deleting says the opposite, that hosttools helps with writing the C++ portions, which I don't know why though.

Reply:
Because in `hosttools` mode the compiler does not bootstrap itself and rebuild all the Swift modules in response to a change in a C++ file. Now that it works with a stable release and we’re almost ready to make it the default, the note applies to Swift changes too.

Reply:
OK, I see, `hosttools` works well for both, got it.

Reply:
Sorry for the confusion. Still learning better ways to express myself.

Reply:
No, you said that above: I just wasn't sure that's what you meant until you went into detail about avoiding the full bootstrap. 😄